recipes/tmpdir/codemod.yml

```
@@ -0,0 +1,21 @@
+schema_version: "1.0"
+name: @nodejs/tmpdir
```

```suggestion
name: "@nodejs/tmpdir"
```

WARNING codemod cli will not parse it if you don't user quote